Woof/Woof/ViewModel/Sitter/SitterProfileViewModel.swift

```diff
@@ -3,7 +3,7 @@ import NetworkService
 
 /// The view model for a pet sitter profile view is responsible for receiving and processing user actions
 /// to change model data and passes back the updated model data.
-final class SitterProfileViewModel: ObservableObject {
+class SitterProfileViewModel: ObservableObject {
```

Comment:
make it final back, as we do not inherit or extend it now.

Reply:
Returned with 4488b64